### PR TITLE
Fixed article search functionality to work with django CMS 3.6

### PR DESCRIPTION
**Pull request by vThaian**
Based on #522 

Checked on 3.4.5, 3.5 and 3.6

### CODE REVIEW

File: aldryn_newsblog/views.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 from datetime import date, datetime
 
+from django.db import OperationalError
```
Comment (FinalAngel) [inline aldryn_newsblog/views.py]: ist his required? I don't see it used anywhere

File: aldryn_newsblog/views.py [hunk 2]
```diff
@@ -262,7 +263,7 @@ class ArticleSearchResultsList(ArticleListBase):
     def get(self, request, *args, **kwargs):
         self.query = request.GET.get('q')
         self.max_articles = request.GET.get('max_articles', 0)
-        self.edit_mode = (request.toolbar and request.toolbar.edit_mode)
+        self.edit_mode = (request.toolbar and toolbar_edit_mode_active(request))
```
Comment (FinalAngel) [inline aldryn_newsblog/views.py]: Does this also still work with django CMS 3.4, I thought thats 3.6+ only?

Comment (vThaian) [reply aldryn_newsblog/views.py]: yes, here is compact: https://github.com/divio/aldryn-newsblog/blob/master/aldryn_newsblog/compat.py#L1

Comment (FinalAngel) [reply aldryn_newsblog/views.py]: Thanks, didn't dig that far 👍

**Pull request merged**